js/models/roomcollection.js

```diff
@@ -28,7 +28,9 @@
 
 	var RoomCollection = Backbone.Collection.extend({
 		model: OCA.SpreedMe.Models.Room,
-		comparator: 'name',
+		comparator: function(model) {
+			return -(model.get('lastPing'));
```

Comment:
Yes, that's right. That attribute is part of `spreedme_room_participants` table.

I'm kind of new with Backbone, so I was just checking what we receive when we do `GET` on `/apps/spreed/api/room` endpoint and assuming all these attributes we receive are part of the RoomCollection Model.